### PR TITLE
update example to use elm-bootstrap package

### PR DESCRIPTION
**Pull request by mathieul**

### CODE REVIEW

**Comment by mccrodp** [comment]
*The Homepage seems to try to fetch a post of some id that doesn't exist?*
screen shot 2017-03-16 at 15 49 35

*Clicking on the link gives a Not found.*
screen shot 2017-03-16 at 15 49 45

File: app/elm/Main.elm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module Main exposing (main)
 
-import Data exposing (Post)
+import Post exposing (Post)
 import Html as H exposing (Html)
 import Html.Attributes as A exposing (class, id)
```
Comment (mccrodp) [inline app/elm/Main.elm]: It seems `id` is no longer needed now :)

File: app/assets/api/posts.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": 1,
+    "title": "Routing using elm-route",
+    "body": "Read more about `Bogdanp/elm-route` at http://package.elm-lang.org/packages/Bogdanp/elm-route/latest"
```
Comment (mccrodp) [inline app/assets/api/posts.json:5]: Can we add markup to make these links clickable?

**Comment by mccrodp** [comment]
While reviewing I came across a separate issue / suggestion: https://github.com/mathieul/brunch-with-elm-bootstrap/issues/6

File: app/elm/Main.elm (hunk 1 above)
Comment (mathieul) [reply app/elm/Main.elm]: True, removed.

File: app/assets/api/posts.json (hunk 2 above)
Comment (mathieul) [reply app/assets/api/posts.json:5]: It is actually trivial doing so with `elm-markdown`. Even though the example is growing by the day, it seems like a worthy addition.

**Pull request merged**